lib/sparkql/parser_tools.rb

```diff
@@ -303,6 +310,23 @@ def arithmetic_type(num1, num2)
     end
   end
 
+  def escape_arithmetic_value(expression)
+    case expression[:type]
+    when :decimal
+      BigDecimal.new(expression[:value])
+    else
+      escape_value(expression)
+    end
+  end
+
+  def unescape_arithmetic(value)
+    if value.is_a?(BigDecimal)
+      value.round(20).to_s('F')
```

Comment:
What's this about?

Reply:
Good find! I concur, we should strive for consistency with postgres versus ruby.